packages/keyring-eth-ledger-bridge/src/ledger-keyring.ts

```diff
@@ -36,6 +34,14 @@ type SignTransactionPayload = Awaited<
   ReturnType<LedgerBridge<LedgerIframeBridgeOptions>['deviceSignTransaction']>
 >;
 
+type AccountPageEntry = {
+  address: string;
+  balance: number | null;
+  index: number;
+};
+
+type AccountPage = AccountPageEntry[];
```

Comment:
Should these be exported?

Reply:
Neither `eslint` nor `tsc` complained about this type not being exported, but IMO we should since it's part of public functions signatures, lemme export them! Thanks!

Reply:
See:
- https://github.com/MetaMask/accounts/pull/55/commits/006878f9776c5ce4810e608175af8ab6883bc42b
- https://github.com/MetaMask/accounts/pull/55/commits/f916f75ce252c5336f995ad77e8d12e0eba56e77